### PR TITLE
feature: versturenEmail policy should check for open zaak

### PR DESCRIPTION
**Pull request by hsiliev**
Restore versturenEmail policy check for open zaak

Solves PZ-2901

### CODE REVIEW

**Review by felixcicatt — APPROVED** [review]
Nice!

File: src/main/java/net/atos/zac/app/mail/MailRestService.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public void sendMail(
             final RESTMailGegevens restMailGegevens
     ) {
         final Zaak zaak = zrcClientService.readZaak(zaakUUID);
-        assertPolicy(policyService.readZaakRechten(zaak).versturenEmail());
+        assertPolicy(policyService.readZaakRechten(zaak).versturenEmailBijAfhandelenZaak());
```
Comment (edgarvonk) [inline src/main/java/net/atos/zac/app/mail/MailRestService.java:76]: Did you check that this REST endpoint is only used in the scenario 'send email when closing zaak'? If it is I think we should renamed this REST endpoint to make that clear. If it is not this policy check is probably not ok here?

Comment (edgarvonk) [reply src/main/java/net/atos/zac/app/mail/MailRestService.java:76]: I mean: it is now a generic endpoint to send email so having this policy check is bound to cause confusion for future developers. It is really weird.

Comment (hsiliev) [reply src/main/java/net/atos/zac/app/mail/MailRestService.java:76]: Yes it is used. It's the only endpoint for sending mail I think, so used for most mail related tasks.

We'll have to create another endpoint and change UI to use it.

Comment (edgarvonk) [reply src/main/java/net/atos/zac/app/mail/MailRestService.java]: If that's the case the current PR as it is actually breaks functionality then maybe?

Comment (hsiliev) [reply src/main/java/net/atos/zac/app/mail/MailRestService.java]: This PR will just restore the versturenEmail policy functionality

**Pull request closed without merging**